core/tee/tee_svc_cryp.c

```diff
@@ -811,7 +851,7 @@ TEE_Result tee_svc_cryp_obj_alloc(TEE_ObjectType obj_type,
 		return res;
 
 	/*
-	 * Verify that maxObjectSize is supported and find out how
+	 * Verify that maxKeySize is supported and find out how
```

Comment:
The parameter `max_obj_size` in this function should be renamed to `max_key_size` to match the comment and the renaming of the renamed field in `TEE_ObjectInfo`.